soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1352,7 +1352,7 @@ std::unordered_map<std::string, RandomizerGet> SpoilerfileGetNameToEnum = {
     { "Acheter: Missiles", RG_BUY_BOMBCHU_5 },
     { "Buy Deku Seeds (30)", RG_BUY_DEKU_SEEDS_30 },
     { "Acheter: Graines Mojo (30)", RG_BUY_DEKU_SEEDS_30 },
-    { "Sold Out", RG_SOLD_OUT },
```

Comment:
just making sure we still have this in english too

Reply:
Whoopsie! Done!